scanapp/tests/test_models.py

```diff
@@ -0,0 +1,260 @@
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        scanned_file = ScannedFile.objects.create(scan=scan, path='/home/nexb/server/')
+        license = License.objects.create(
+            scanned_file=scanned_file, 
+            key='A',
+            score=78,
+            short_name='mit',
+            category='some category',
+            owner='mit',
+            homepage_url='https://github.com/',
+            text_url='http://github.com/',
+            dejacode_url='https://github.com',
+            spdx_license_key='mit', 
+            spdx_url='https://github.com/',
+            start_line=21,
+            end_line=567,
+            matched_rule=json.loads('{"url": ["https://github.com", "https://google.com"]}')
+        )
+        self.assertEqual('mit', license.owner)
+        self.assertEqual(license.key, str(license))
+        self.assertEqual('licenses', license._meta.verbose_name_plural)
+
+
+class CopyrightTestCase(TestCase):
+    def test_copyright_added(self):
+        user = User.objects.create_user(username='username', password='password')
+        scan = Scan.objects.create(
+            user=user,
+            url='https;//github.com',
+            scan_directory='media/username',
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        scanned_file = ScannedFile.objects.create(scan=scan, path='/home/nexb/server/')
+        copyright = Copyright.objects.create(
+            scanned_file=scanned_file,
+            start_line=800,
+            end_line=1000
+        )
+        self.assertEqual(1000, copyright.end_line)
+        self.assertEqual(str(copyright.start_line), str(copyright))
+        self.assertEqual('copyrights', copyright._meta.verbose_name_plural)
+
+
+class CopyrightHolderTestCase(TestCase):
+    def test_copyright_holder_added(self):
+        user = User.objects.create_user(username='username', password='password')
+        scan = Scan.objects.create(
+            user=user,
+            url='https;//github.com',
+            scan_directory='media/username',
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        scanned_file = ScannedFile.objects.create(scan=scan, path='/home/nexb/server/')
+        copyright = Copyright.objects.create(
+            scanned_file=scanned_file,
+            start_line=800,
+            end_line=1000
+        )
+        copyright_holder = CopyrightHolder.objects.create(copyright=copyright, holder='mit')
+        self.assertEqual('mit', copyright_holder.holder)
```

Comment:
You need to get the object you have saved ... otherwise you are not testing anything.... And this applies to most of your model tests